api/src/main/java/org/openmrs/module/ugandaemrreports/reports/SetupANCRegister.java

```diff
@@ -198,43 +193,56 @@ private DataSetDefinition dataSetDefinition() {
         //start constructing of the dataset
         PersonAttributeType attribute = Context.getPersonService().getPersonAttributeTypeByUuid("14d4f066-15f5-102d-96e4-000c29c2a5d7");
```

Comment:
@ningosi 
General observations
All ages are supposed be calculated based on the period the report is run and not based on the current date. See AgeLimitCalculation.